### DISCUSSION TITLE
Where is the data of the app stored?

### DISCUSSION THREAD

**Discussion by Ali-Riese** [te-online/timemanager#137] [General]
Hey, 
i have a complete new setup of nextcloud on an RPI4 and was wondering if i can restore the data of an other Instance that i have backuped?
I have a complete backup of the data directory of the "old" instance.

Is there any possible way to get the clients, projects, etc. from the backup into the new instance? 

Best, Ali

**Comment by te-online** [comment]
Hey @Ali-Riese 

It's a blessing and a curse – the data is stored in the database. If you have an SQL backup of your old instance, you can copy the tables prefixed with `timemanager_` into your new database. The only thing you may need to adjust are the user ids in case you created different user accounts on your new instance.

Hope this helps!

**Reply by Ali-Riese** [reply]
Thanks @te-online for the quick answer.

This is exactly what i have needed to know.
